client/account-recovery/lost-password-form/index.jsx

```diff
@@ -79,30 +84,32 @@ export class LostPasswordFormComponent extends Component {
 					</p>
 				</Card>
 				<Card compact>
-					<FormLabel>
-						{ translate( 'Username or Email' ) }
+					<form onSubmit={ this.submitForm }>
+						<FormLabel>
+							{ translate( 'Username or Email' ) }
 
-						<FormInput
-							className="lost-password-form__user-login-input"
-							onChange={ this.onUserLoginChanged }
-							value={ userLogin || '' }
-							disabled={ isRequesting } />
-					</FormLabel>
-					{
-						requestError && (
-						<p className="lost-password-form__error-message">
-							{ translate( 'We encountered some problems with that login information. ' +
-								'Please provide another one or try again later.' ) }
-						</p> )
-					}
-					<Button
-						className="lost-password-form__submit-button"
-						onClick={ this.submitForm }
-						disabled={ isPrimaryButtonDisabled }
-						primary
-					>
-						{ translate( 'Get New Password' ) }
-					</Button>
+							<FormInput
+								className="lost-password-form__user-login-input"
+								onChange={ this.onUserLoginChanged }
+								value={ userLoginFormValue }
+								disabled={ isRequesting } />
+						</FormLabel>
+						{
+							requestError && (
+							<p className="lost-password-form__error-message">
+								{ translate( 'We encountered some problems with that login information. ' +
```

Comment:
I saw this error when I typed an incorrect email address but found the message a bit confusing, as if there was an error out of my control. Could we be more specific here?